scripts/set_reviewers.py

```diff
@@ -0,0 +1,139 @@
+import datetime
```

Comment:
Would be nice to set this up for local development and unit testing. What do you think about moving it to a dedicated directory?

```
owners
 --teams.json
 --assignments.json
 --set_reviewers.py
 --test_set_reviewers.py
 --requirements.txt
 --README.md
```

Reply:
otherwise do you want to add a `test_set_reviewers.py`? IMO tests make it a little easier to keep track of what the code is supposed to do

Reply:
Done